src/components/Step/hooks/use-step.spec.js

```diff
@@ -354,36 +354,4 @@ describe('useStep', () => {
 
     unmount();
   });
-
-  it('should support onPlay action handler props', () => {
```

Comment:
Why are we removing the spec specifically?

Reply:
It was duplicated. The updated rules in Eslint caught it :)